staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -86,8 +86,11 @@ func newETCD3HealthCheck(c storagebackend.Config) (func() error, error) {
 		client := clientValue.Load().(*clientv3.Client)
 		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
 		defer cancel()
+		// Try and read a key from etcd, if we can retrieve that key, then we can establish:
+		// 		1. that we can talk to our local etcd
```

Comment:
The etcd could be local or remote right ?

Reply:
Yeah, it depends on the configuration.

Reply:
*Nit comment : 
So do we think we should change the comment ?